src/components/DatePicker/DatePicker.vue

```diff
@@ -13,23 +13,24 @@
             min-width="290"
     >
       <template v-slot:activator="{ on }">
-        <v-text-field id="date-text-field"
-                      ref="dateTextField"
-                      append-icon="mdi-calendar"
-                      autocomplete="chrome-off"
-                      :error-messages="errorMsg"
-                      :error="!!errorMsg"
-                      :value="dateText"
-                      :label="title"
-                      :name="Math.random()"
-                      :rules="inputRules"
-                      :disabled="disablePicker"
-                      v-on="on"
-                      v-on:keydown="$event.preventDefault()"
-                      v-on:keyup.enter="emitDate(dateText)"
-                      v-on:click:append="on.click"
-                      filled
-        />
+        <span :class="{'date-text-field-pointer': enableSelector}" v-on="enableSelector && on">
```

Comment:
Consider doing a tech demo to explain how v-on works. (I'd like to know 😁 )

Reply:
Great idea! I'll have to do some more research myself 😆